tests/wasm/mod.rs

```diff
@@ -18,27 +19,84 @@ use rustls::version::TLS13;
 use rustls::Certificate;
 use tempfile::{tempdir, NamedTempFile};
 
+#[cfg(enarx_with_shim)]
+fn with_signatures(cmd: &mut Command) -> &mut Command {
```

Comment:
Github is showing some symbols that are only used in here that produce unused symbol warnings on Mac and Windows, please move those imports into the function.